include/boost/process/detail/windows/locale.hpp

```diff
@@ -39,14 +40,15 @@ class windows_file_codecvt
      const char* from, const char* from_end, const char*& from_next,
      wchar_t* to, wchar_t* to_end, wchar_t*& to_next) const override
    {
+     boost::ignore_unused(state);

```

Comment:
Better style or not, that's not the point, the point is to fix the warning. Remove the parameter or mark it as unused.